qa/suites/fs/upgrade/mds_upgrade_sequence/tasks/0-from/v17.2.0.yaml

```diff
@@ -4,27 +4,20 @@ teuthology:
 
 meta:
 - desc: |
-   setup ceph/pacific v16.2.4
```

Comment:
This special case for v16.2.4 was added due to
```
commit dcda5cb9ce52a1d194a461d332f6ce497ef19586
Author: Patrick Donnelly <pdonnell@redhat.com>
Date:   Tue Nov 16 15:04:59 2021 -0500

    qa: disable metrics on kernel client during upgrade
    
    v16.2.4 MDS triggers an assert from these messages.
    
    Also: add latest pacific for extra coverage.
```

This yaml can be done away with I guess.

Reply:
yeah, just the `quincy.yaml` is enough

Reply:
removed the v17.2.0 yaml